_partials/about-device-reset.md

```diff
@@ -1,6 +1,11 @@
 <!--- Note about the device reset. Applies to both Home Assistant Green and Yellow. -->
 
-> **Note**
-> This process clears the data disk on your device. Unless you create a backup, you will lose all data and configuration settings.
-> A device reset wipes the data partition. It removes all user data as well as [Home Assistant Core](https://www.home-assistant.io/docs/glossary/#home-assistant-core), [Supervisor](https://www.home-assistant.io/docs/glossary/#home-assistant-supervisor), and any installed add-ons. The [Home Assistant Operating System](https://www.home-assistant.io/docs/glossary/#home-assistant-operating-system) stays on the system.
-> Once a device wipe is initiated, the data partition is marked as to be cleared on boot. Then, the system reinstalls the latest version of all Home Assistant components.
+
+{% hazard "notice" "**Risk of data loss**. This process clears the data disk on your device.
```

Comment:
Is there a way to structure this notice a bit?
![image](https://github.com/user-attachments/assets/ca5c0b43-c595-4356-9033-a11b23419910)